external-crates/move/crates/move-package-alt/src/errors/files.rs

```diff
@@ -108,6 +108,12 @@ impl Debug for FileHandle {
     }
 }
 
+impl Display for FileHandle {
```

Comment:
I added this for printing out paths in error messages, but then I rolled that back because of path snapshot problems.